plugins/sentinelone/komand_sentinelone/actions/run_remote_script/action.py

```diff
@@ -0,0 +1,73 @@
+import insightconnect_plugin_runtime
+from insightconnect_plugin_runtime.exceptions import PluginException
+from .schema import RunRemoteScriptInput, RunRemoteScriptOutput, Input, Output, Component
+from komand_sentinelone.util.constants import (
+    SCRIPT_TIMEOUT_LOWER_LIMIT,
+    SCRIPT_TIMEOUT_UPPER_LIMIT,
+    SCRIPT_TIMEOUT_DEFAULT,
+)
+from komand_sentinelone.util.helper import check_password_meets_requirements
+
+# Custom imports below
+
+
+class RunRemoteScript(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="run_remote_script",
+            description=Component.DESCRIPTION,
+            input=RunRemoteScriptInput(),
+            output=RunRemoteScriptOutput(),
+        )
+
+    def run(self, params={}):
+        agent_ids = params.get(Input.IDS)
+        script_id = params.get(Input.SCRIPT_ID, "")
+        task_description = params.get(Input.TASK_DESCRIPTION, "")
+        script_timeout = params.get(Input.TIMEOUT, SCRIPT_TIMEOUT_DEFAULT)
+        input_params = params.get(Input.INPUT_PARAMETERS, "")
+        password = params.get(Input.PASSWORD, "")
+
+        if not script_id:
```

Comment:
Can't we just set `required: true` in the plugin spec for this so that the user **has** to provide it?

Reply:
They are already required so I've removed this check.